src/lib/dataEntities/networks.ts

```diff
@@ -178,6 +179,7 @@ export const l2Networks: L2Networks = {
     confirmPeriodBlocks: 45818,
     rpcURL: process.env['ARB_ONE_RPC'] || 'https://arb1.arbitrum.io/rpc',
     isCustom: false,
+    retryableLifetimeSeconds: 7 * 24 * 60 * 60
```

Comment:
could create a `const SEVEN_DAYS_IN_SECONDS = 7 * 24 * 60 * 60` to make this clearer